crates/burn-import/src/burn/node/random_uniform.rs

```diff
@@ -0,0 +1,67 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, Type};
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone)]
+pub struct RandomUniformNode {
+    pub high: f64,
+    pub low: f64,
+    pub output_ty: TensorType,
+}
+
+impl RandomUniformNode {
+    pub fn new(output_ty: TensorType, high: f64, low: f64) -> Self {
+        Self {
+            high,
+            low,
+            output_ty,
+        }
+    }
+
+    fn get_output_shape(&self) -> TokenStream {
+        let shape_it = self
+            .output_ty
+            .shape
+            .as_ref()
+            .expect("RandomUniform output has no shape!")
+            .iter();
+        quote! { Shape::new([#(#shape_it),*]) }
+    }
+
+    fn get_distribution(&self) -> TokenStream {
+        let low = self.low;
+        let high = self.high;
+        quote! { Distribution::Uniform(#low, #high) }
+    }
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for RandomUniformNode {
+    fn input_types(&self) -> Vec<Type> {
+        Vec::with_capacity(0)
+    }
+
+    fn output_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.output_ty.clone())]
+    }
+
+    fn forward(&self, _scope: &mut Scope, _node_position: usize) -> TokenStream {
+        let output = &self.output_ty.name;
+        let shape = self.get_output_shape();
+        let dist = self.get_distribution();
+        quote! {
+            let #output = Tensor::random(#shape, #dist, self.device.deref());
```

Comment:
`Tensor::random` takes a reference to the device iirc, so I think it should be `&self.device` no?

Reply:
This stems from the fact that self.device is not directly a `Backend::Device`, but rather a `module::Ignored<Backend::Device>`:

```rust
45 |           let randomuniform1_out1 = Tensor::random(
   |  ___________________________________^
46 | |             Shape::new([2usize, 3usize]),
47 | |             Distribution::Uniform(0f64, 1f64),
48 | |             &self.device,
49 | |         );
   | |_________^ expected associated type, found `Ignored<<B as Backend>::Device>`
   |
   = note: expected associated type `<B as burn::prelude::Backend>::Device`
                       found struct `Ignored<<B as burn::prelude::Backend>::Device>`
```

Alternatively, `&*self.device` would work and also not require importing the Deref trait - personally, I find it looks a bit cleaner with the explicit deref, but have no hard feelings either way, so I'll change it.

Reply:
Oh! Totally forgot about the newly added `Ignored` type wrapper. Never mind that comment then :)